test/cp.js

```diff
@@ -112,6 +112,32 @@ test('dest already exists', t => {
   t.is(shell.cat('resources/file2').toString(), oldContents);
 });
 
+test('-nR does not overwrite an existing file at the destination', t => {
+  // Create tmp/new/cp/a
+  const dest = `${t.context.tmp}/new/cp`;
```

Comment:
Actually, it looks like one of the `mv` tests failed on windows for this exact reason. Maybe we should create another issue for this?